.github/workflows/lib-injection.yml

```diff
@@ -11,6 +11,17 @@ jobs:
     - name: Set up QEMU
       uses: docker/setup-qemu-action@v2
 
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+    # required for getting the version
+    - run: pip install setuptools-scm
+    - name: Get library version
+      id: get_version
+      run: |
+        DDTRACE_PYTHON_VERSION=$(scripts/get_install_version.py)
```

Comment:
Could we change the name of the env var to `DDTRACE_PYTHON_LIB_VERSION` to make clear it's not talking about the python version?

Reply:
I have no strong feeling, this environment variable isn't public at all, only used during the docker build.